src/err.js

```diff
@@ -0,0 +1,19 @@
+function errorCatch(text, toCase) {
+  const errors = [];
+
+  if (!text) {
+    errors.push({ message: 'Text to convert is required' });
```

Comment:
You can't pass the tests with this messages
Make a separate object with your all errors messages